doc/conf.py

```diff
@@ -52,15 +96,6 @@
 copyright = u'2017, Mozilla Research'
 author = u'Mozilla Research'
```

Comment:
Should probably update copyright and author here too.

Reply:
Sure, what do you want here?

Reply:
Matching above, so 2019 Mozilla Corporation

Reply:
Indeed that part is just useless, it's overwriting previous values